src/scripts.js

```diff
@@ -97,6 +118,69 @@ function getUserById(id, listRepo) {
   return listRepo.getDataFromID(id);
 }
 
+
+function windowOnClick(event) {
+  if (event.target.classList.contains("new-hydration-button")) {
+    
+    submitNewHydration(userNowId);
+  }
+  if (event.target.classList.contains("new-activity-button")) {
+    submitNewActivity(userNowId);
+  }
+  if (event.target.classList.contains("new-sleep-button")) {
+    submitNewSleep(userNowId);
+  }
+}
+
+function getTodaysDate() {
+  var today = new Date();
+  var dd = String(today.getDate()).padStart(2, '0');
+  var mm = String(today.getMonth() + 1).padStart(2, '0'); //January is 0!
+  var yyyy = today.getFullYear();
+  return today = yyyy + '/' + mm + '/' + dd;
+}
+
+
+function submitNewHydration(userNowId) {
+  let todaysDate = getTodaysDate()
+  let newHydrationValue = newHydrationInput.value
+  if (!newHydrationValue) {
+    return alert("Please enter a value")
+  } else if (newHydrationValue > 128) {
+    newHydrationValue = 128
+  }
+  let postedHydration = requests.postHydrationData(userNowId, todaysDate, newHydrationValue)
+  Promise.all([postedHydration])
+    .then(value => {
+      updatePageHydration (userNowId, todaysDate) 
+  });
+  updateHydrationButton.disabled = true;
+}
+
+
+function updatePageHydration (userNowId, todaysDate) {
+  let newRecievedHydration = requests.fetchHydrationData()
+  Promise.all([newRecievedHydration])
+    .then(value => {
+      hydrationData = value[0];
+      // let hydrationElementsToClear = q
+      hydrationToday.innerHTML = '';
+      startApp(userNowId)
+      // addDailyOuncesInfo(userNowId, currentHydrationRepo, todaysDate);
+    })
+}
+
+function submitNewActivity(userNowId) {
```

Comment:
I like that you set these up to work on in the future ... which we have now done and will have in one of our next PRs!